webapps/ui/cockpit/src/plugins/cockpit.processInstance.runtime.tab/variables/Variables.js

```diff
@@ -0,0 +1,278 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useCallback, useEffect, useState } from "react";
+import withProcessInstance from "../../../components/ProcessInstance/HOC/withProcessInstance";
+import withBpmn from "../../../components/ProcessInstance/HOC/withBpmn";
+import withActivityInstances from "../../../components/ProcessInstance/HOC/withActivityInstances";
+import translate from "utils/translation";
+import { post, get } from "utils/request";
+import { formatDate } from "utils/formatting";
+import { getItem, setItem } from "utils/localstorage";
+import { getSearchPillsConfig } from "./SearchPillsConfig";
+import { SearchPills, TablePaginated, Table, Clipboard } from "components";
+
+import "./Variables.scss";
+
+const ROWS_PER_PAGE = 50;
+const PAYLOAD = {};
+
+const header = [
+  { searchQuery: "variableName", label: "PLUGIN_VARIABLE_NAME" },
+  { searchQuery: "variableType", label: "PLUGIN_VARIABLE_TYPE" },
+  { label: "PLUGIN_VARIABLE_VALUE" },
+  { label: "PLUGIN_VARIABLE_SCOPE" }
```

Comment:
I'd like to use `translate` at the position where we use the translate string. This way, we don't have to think about whether or not we need to translate strings from Objects later.

What do you think about this? Should we add this to the best practices page?